app/jobs/record_usages.py

```diff
@@ -161,6 +219,7 @@ def record_user_usages():
 
     for node_id, params in api_params.items():
         record_user_stats(params, node_id, usage_coefficient[node_id])
+        record_admin_stats(params, node_id, usage_coefficient[node_id])
```

Comment:
separate record_admin_stats and record_user_stats , they both use DISABLE_RECORDING_NODE_USAGE variable

Reply:
i think its a general variable it is also used for record_node_usages()

Reply:
yeap it is and it's better to become separated for each one